### PR TITLE
basic integration test with some changes

### PR DESCRIPTION
**Pull request by achang1**

### CODE REVIEW

File: uwuno/src/main/java/com/learning/uwuno/controllers/playerController.java [hunk 1]
```diff
@@ -87,18 +87,18 @@ else if (parser.exists("draw")) {
...
                 parser.exists("cardColor") &&
                 parser.exists("cardValue") &&
-                parser.exists("wildColor")) {
+                parser.exists("setWildColor")) {
             return ResponseEntity.ok(containerService.playCard(uid, pid,
                                         parser.getValue("cardType"),
                                         parser.getValue("cardColor"),
                                         parser.getValue("cardValue"),
-                                        parser.getValue("wildColor")));
+                                        parser.getValue("SetWildColor")));
```
Comment (rico-law) [inline uwuno/src/main/java/com/learning/uwuno/controllers/playerController.java]: Wrong capitalization

Comment (achang1) [reply uwuno/src/main/java/com/learning/uwuno/controllers/playerController.java]: i cri

File: uwuno/src/main/java/com/learning/uwuno/services/gameService.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
     // Class Functions
     // POSTS
     public room addRoom(String roomName, boolean useBlankCards) {
+        if (roomName.isEmpty()) throw new badRequest();
```
Comment (rico-law) [inline uwuno/src/main/java/com/learning/uwuno/services/gameService.java]: :(

Comment (achang1) [reply uwuno/src/main/java/com/learning/uwuno/services/gameService.java]: nani i swear i changed it

Comment (achang1) [reply uwuno/src/main/java/com/learning/uwuno/services/gameService.java]: curse merge conflictttt

File: uwuno/src/test/java/com/learning/uwuno/integration/tests/basicPlayerTest.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
+//package com.learning.uwuno.integration.tests;
```
Comment (rico-law) [inline uwuno/src/test/java/com/learning/uwuno/integration/tests/basicPlayerTest.java:1]: Commit for accident?

Comment (achang1) [reply uwuno/src/test/java/com/learning/uwuno/integration/tests/basicPlayerTest.java:1]: i'll add it later. lol just ignore for now

**Review by rico-law — APPROVED** [review]

**Pull request merged**